modules/ROOT/pages/shared-resources.adoc

```diff
@@ -208,41 +208,47 @@ By default, when you deploy either an application associated with a domain or a
 [NOTE]
 Mule domain projects cannot be installed using Runtime Manager.
 
-To set this in Studio, open the drop-down menu next to the play button and select *Run Configurations*.
+=== Deploy from Anypoint Studio
 
-image::run-configurations-1.png[run+configurations+1]
+The following steps describe how to deploy your domain project and the associated applications from Studio:
 
-Then pick the *General* tab, and tick or untick the boxes next to the projects that you want to always deploy together with the application that is currently selected on the navigation menu to the right.
+. Open the drop-down menu next to the play button and select *Run Configurations*.
```

Comment:
Is it clear which button is the play button? Is it labeled Play? Or does it have an icon on it?

Reply:
And just ignore me if the image makes this clear!